lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -3902,10 +3901,19 @@ def get_extended_due(course, unit, user):
         if text_type(override['location']) == location:
             return override['actual_date']
     print(unit.location)
-    print(dates)
+    print(list(dates))
```

Comment:
how come we want print statements here?

Reply:
Great question. They look like they are designed to help you if your test is failing. They predate me, I was just fixing them to actually work (it was printing "<generator>" before).

I could also remove them?

Reply:
Removed.